interface/forms/eye_mag/php/taskman_functions.php

```diff
@@ -197,7 +197,7 @@ function deliver_document($task)
         return true;
     } else {
         $email_status = $mail->ErrorInfo;
-        error_log("EMAIL ERROR: ".$email_status, 0);
+        error_log("EMAIL ERROR: ".errorLogEscape($email_status, 0));
```

Comment:
the second parameter of error_log should not be nested in errorLogEscape. should instead be:
```
error_log("EMAIL ERROR: ".errorLogEscape($email_status), 0);
```

Reply:
Oops, I made the same mistake twice.  Thank you!!